Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -906,18 +896,5 @@ public virtual void OnClientSceneChanged(NetworkConnectionToServer conn)
 
         #endregion
 
-        #region Start & Stop callbacks
```

Comment:
beautiful!

Reply:
it was very satisfying. wish you could have been there.